File: cpp/problems/MultiplicationTable.cpp
#include <iostream>
using namespace std;
int main()
{
    cout << "Enter the number to print the multiplication table: "; // ask user to enter number
    int num; // number to print multiplication table
    cin >> num; // take input from user
    for (int i = 1; i <= 10; i++)
    {
        cout << num << " * " << i << " = " << num * i << endl; // print multiplication table
    }
    return 0; //
}

File: go/problems/multiplicationTable.go
package main

import "fmt"

func multiplicationTable() {
	var num int // Declare a variable to hold the number for which we want to print the multiplication table
	fmt.Print("Enter the number to print the multiplication table: ")
	fmt.Scan(&num) // Read the number from user input
	for i := 1; i <= 10; i++ {
		fmt.Printf("%d * %d = %d\n", num, i, num*i)
	}
}
